.github/workflows/run-tests.yml

```diff
@@ -220,18 +220,14 @@ jobs:
           TEST_LOG_PATH: ${{ github.workspace }}/artifacts/log/test-logs
           TestsRunningOutsideOfRepo: true
         run: >
-          dotnet test -s .runsettings -v:n ${{ env.TEST_ASSEMBLY_NAME }}.dll
-          -l "console;verbosity=normal"
-          -l "trx;LogFilePrefix=${{ inputs.testShortName }}"
-          -l "GitHubActions;summary.includePassedTests=true;summary.includeSkippedTests=true"
-          --blame
-          --blame-hang-timeout ${{ inputs.testHangTimeout }}
-          --blame-crash
+          dotnet exec ${{ env.TEST_ASSEMBLY_NAME }}.dll
```

Comment:
Is this really how normal developers are supposed to execute tests using the Microsoft.Testing.Platform? In general, we don't suggest to people to use `dotnet exec`. You can `dotnet XXX.dll` or just run the platform-specific executable `${{ env.TEST_ASSEMBLY_NAME }}[.exe]`

Reply:
What's the preference from the test team?

I guess I would prefer invoking the executable directly.

Reply:
I'm reworking the infra with expectation that we'd be running `build.cmd/sh` for all build, test, package, etc. needs.

Reply:
That's not a correct expectation, as for nuget testing, and tests run on helix we run just the binaries, and thus no arcade or repository is available.

Reply:
We won't be running on Helix at all.

Reply:
We are using `dotnet exec` because the tests are being run without the project file available. But since the test assembly is executable now, we can directly run that.

Reply:
I tried to move over to running executable directly, but it's not in the zip file when using `ZipTestArchive`. I have no idea why. Maybe we are zipping too early or something. I'm doing dotnet test.dll for now. We can follow-up with investigation to include the exe in the zip.